### PR TITLE
Update Gradle cache version to jars-8

### PR DESCRIPTION
**Pull request by starsep**
I have noticed that I do not have directory `~/.gradle/caches/jars-3` locally.
On my computer `ls ~/.gradle/caches/ | grep jars` returns jars-8 only.

Reference: https://github.com/gradle/gradle/blob/1e3ce9cf4394e3e0e290ef4a647ffb5e0d1343dc/subprojects/core/src/main/java/org/gradle/internal/classpath/DefaultClasspathTransformerCacheFactory.java#L46

All samples use Gradle 6.6 at the moment:

`grep -R distributionUrl`

> Owl/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
Jetsnack/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
JetNews/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
Jetchat/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
Jetsurvey/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
Crane/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
Rally/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
Jetcaster/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip

### CODE REVIEW

**Comment by JoseAlcerreca** [comment]
Good c̶a̶c̶h̶e̶ catch! I'm going to let Chris merge this one.

File: .github/workflows/Crane.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
         with:
           path: |
             ~/.gradle/caches/modules-2
```
Comment (chrisbanes) [inline .github/workflows/Crane.yaml:39]:
```suggestion
            ~/.gradle/caches/modules-*
```

File: .github/workflows/Crane.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ jobs:
...
-            ~/.gradle/caches/jars-3
+            ~/.gradle/caches/jars-8
```
Comment (chrisbanes) [inline .github/workflows/Crane.yaml:40]:
```suggestion
            ~/.gradle/caches/jars-*
```

File: .github/workflows/Crane.yaml [hunk 3]
```diff
@@ -37,7 +37,7 @@ jobs:
...
             ~/.gradle/caches/build-cache-1
```
Comment (chrisbanes) [inline .github/workflows/Crane.yaml:41]:
```suggestion
            ~/.gradle/caches/build-cache-*
```

**Review by chrisbanes — CHANGES_REQUESTED** [review]
Let's just use wildcards instead. @starsep are you happy to copy these changes to the other samples? 

No worries if not, I'll do it.

**Review by chrisbanes — APPROVED** [review]

**Review by JoseAlcerreca — APPROVED** [review]

**Pull request merged**